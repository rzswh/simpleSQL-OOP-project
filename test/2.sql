CREATE DATABASE qaz;
CREATE DATABASE edc;
CREATE DATABASE ijn;
SHOW DATABASES;
CREATE DATABASE acs;
SHOW DATABASES;
DROP DATABASE acs;
DROP DATABASE qaz;
DROP DATABASE edc;
DROP DATABASE ijn;
