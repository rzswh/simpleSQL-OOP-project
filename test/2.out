Database
edc
ijn
qaz
Database
acs
edc
ijn
qaz
